css/styles.css

```diff
@@ -218,6 +242,130 @@ i {
   opacity: 0.85;
 }
 
+.error-message-data {
+  background-color: white;
+  box-shadow: 0 2px 5px 1px rgb(15, 15, 15);
+  border-radius: 12px;
+  z-index: 9999;
+}
+
+.reload-button-div {
+  text-align: center;
+}
+
+.reload-button {
+  color: white;
+  background-color: var(--heart-color);
+  border-radius: 12px;
+  transition: transform 200ms;
+  box-shadow: 0 2px 5px 0.5px rgb(160, 160, 160);
+}
+
+.reload-button:hover,
+.reload-button:active {
+  transform: scale(1.2);
+}
+
+/* LOADING SPINNER */
+
+.lds-spinner {
+  color: official;
+  display: inline-block;
+  position: fixed;
+  top: 50%;
+  left: 39%;
+  width: 80px;
+  height: 80px;
+  z-index: 9999;
```

Comment:
Why the astronomical `z-index`? Even if this is necessary, I think you could choose something like 5, there's no need to go overboard.